language-design.md

```diff
@@ -48,8 +58,11 @@ How do the themes of _Growing a Language_ relate to the "sound lab" we did this
 In what way is an API a language? 
 
 **Response**
+APIs can be categorized as internal domain-specific languages, since they are written inside an existing host language.
 
+Most of the APIs are designed with consistency and symmetry, just like any other programming languages since "every API is a little language, and people must learn to read and write it" (Bloch, 2006). This should help create a feeling of fluency that most languages possess. By having this characteristic of languages, APIs make it easy for users to detect patterns to write new code from a similar section.
```

Comment:
May you expand what you mean by symmetry?

In some API's the public facing functions are often times abstracting complex, coordinated calls to other services and functions&mdash;in these cases I don't image there existing a symmetry between the API abstraction and the source.